### PR TITLE
fixes a few grammatical errors, clarifies a few things

### PR DESCRIPTION
**Pull request by drammock**
This one is auto-mergable. have a look @bambooforest, and merge if you're happy.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ Ideally, every phonological inventory in PHOIBLE would:
 2. describe all consonants, vowels, and tonemes
 3. include information about major allophonic alternations
 
-However, because PHOIBLE subsumes information from several pre-existing compilations of phonological inventories, not all inventories fully comply with the second and third criteria. Violations of the second criterion are limited to inventories that describe consonants and vowels but not tones; violations of the third criterion are discussed more fully below.
+However, because PHOIBLE subsumes information from several pre-existing compilations of phonological inventories, not all inventories fully comply with these criteria. Violations of the second criterion are limited to inventories that describe consonants and vowels but not tones; violations of the third criterion are discussed more fully below.
```
Comment (drammock) [inline index.md:20]: Here I'm trying to leave open the possibility of unpublished data :)

File: index.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ Ideally, every phonological inventory in PHOIBLE would:
...
 
-If you're interested in analyzing allophones, a further issue is that there is no standard for what counts as a “major” allophonic alternation, and levels of allophonic detail will necessarily vary from author to author. In all cases, if a phonological description includes *some* information about allophonic alternations, it is assumed that phonemes with no allophones mentioned do not exhibit “major” allophonic variation (in the judgment of the original author(s) of the language description). For such phonemes, the sole allophone is treated as identical with the phonemic representation.
+If you’re interested in analyzing allophones, a further issue is that there is no standard for what counts as a “major” allophonic alternation, and levels of allophonic detail will necessarily vary from author to author. In all cases, if a phonological description includes *some* information about allophonic alternations, it is assumed that phonemes with no allophones mentioned do not exhibit “major” allophonic variation (in the judgment of the original author(s) of the language description). For such phonemes, the sole allophone is treated as identical with the phonemic representation.
```
Comment (drammock) [inline index.md:24]: This is just a straight-quote to smart-quote change on "you're"

File: conventions.md [hunk 3]
```diff
@@ -85,7 +86,7 @@ The representations of speech sounds in PHOIBLE do not make use of the tie bar (
...
+Each representation of a phoneme is first normalized into a canonical decomposition form that adheres to the [Unicode Normalization Form D (NFD)](http://unicode.org/reports/tr15/). However, the NFD algorithm does not define a prescribed order for all possible pairs of combining characters. For example, ==0301== (combining acute accent) and ==0308== (combining diaresis) are not re-ordered with respect to one another by the NFD algorithm, regardless of which order they occurred in the input string. PHOIBLE imposes an ordering that is consistent with NFD, but sets an explicit order for all combining characters used in the database. A rigid ordering for spacing modifier letters is also enforced. The chosen ordering is guided by the linguistic literature and typographical concerns, since to the best of our knowledge the IPA does not provide a recommended ordering for diacritics.
```
Comment (drammock) [inline conventions.md:89]: Change is from "Each segment type that is composed of more than one character..." to "Each representation of a phoneme..."  I did this for 2 reasons: 
1. we don't define "segment type" anywhere, and I think it's sort of a legacy term that we don't need to maintain, and
2. we actually denormalize everything, not just strings comprising more than one character.  In fact, that is the whole reason that c-cedilla is a pain in our butts.

File: conventions.md [hunk 4]
```diff
@@ -35,12 +36,12 @@ Languages described as having a sibilant/non-sibilant distinction among coronal
 Retroflex clicks are represented by `‼` (==203C DOUBLE EXCLAMATION MARK==). Note that `ǃ` (==01C3==, the symbol representing (post)alveolar clicks in the IPA) is confusingly referred to as ==LATIN LETTER RETROFLEX CLICK== in the Unicode standard.
 
 ## Glottalization
-Sounds described as “glottalized” are marked with `ˀ` (==02C0 MODIFIER LETTER GLOTTAL STOP==), unless it is clear from context that either “ejective” or “creaky voicing” are the intended meaning. See the section on [Glottalization, laryngealization, and creaky voicing](#glottalization-laryngealization-and-creaky-voicing) for more details.
+Sounds described as “glottalized” are marked with `ˀ` (==02C0 MODIFIER LETTER GLOTTAL STOP==), unless it is clear from context that either “ejective” or “creaky voicing” are the intended meaning. See the section on [Glottalization, laryngealization, and creaky voicing](#glottalization,-laryngealization,-and-creaky-voicing) for more details.
```
Comment (drammock) [inline conventions.md:39]: trying to get the internal link to work. Try this and see, we might have to replace the commas with `-`.

File: conventions.md [hunk 5]
```diff
@@ -24,6 +24,7 @@ Languages described as having a fortis/plain/lenis distinction that corresponds
 
 ## Frictionalized sounds
 The frictionalized diacritic ~~`͓`~~ (==0353 COMBINING X BELOW==) is used in PHOIBLE to represent three types of sounds:
+
```
Comment (drammock) [inline conventions.md:27]: list needed a blank line before to render properly.

File: index.md (hunk 1 above)
Comment (bambooforest) [inline index.md:20]: Thanks!

File: conventions.md (hunk 3 above)
Comment (bambooforest) [inline conventions.md:89]: I agree. I've been asked this question several times, i.e. what is a segment. I also think this is a very clear formulation of the problems we've faced with (de)composition.

File: conventions.md (hunk 4 above)
Comment (bambooforest) [inline conventions.md:39]: @drammock -- link doesn't work for me, but also doesn't fail completely -- it goes to the conventions home page :)

**Pull request merged**